src/Adapter/Debug/DebugModeConfiguration.php

```diff
@@ -90,16 +119,37 @@ public function updateConfiguration(array $configuration)
         $errors = [];
 
         if ($this->validateConfiguration($configuration)) {
+            // Set configuration
             $this->configuration->set('PS_DISABLE_NON_NATIVE_MODULE', $configuration['disable_non_native_modules']);
             $this->configuration->set('PS_DISABLE_OVERRIDES', $configuration['disable_overrides']);
 
             $this->classIndexCacheClearer->clear();
 
-            $status = $this->updateDebugMode((bool) $configuration['debug_mode']);
+            // Update Module Status
+            $modules = [];
+            $filters = (new AddonListFilter())
```

Comment:
What a complex way to retrieve modules 😅  (not your fault I just comment)

Reply:
This is clearly something we will have to remove, the AddonListFilter is malformed :/